src/test/server_test.cpp

```diff
@@ -342,3 +342,42 @@ TEST(Server, ProperShutdownInCaseOfStartError) {
     t.join();
     // this test should not hang
 }
+
+TEST(Server, grpcArguments) {
```

Comment:
I think it would be better to use ovmsconfig_test.cpp to check parsing of new argumenst. It makes sense to start server and pay time in UT's only if we actually verify that those settings actually change ovms behavior.

Either we push those kind of tests to functional, or we change connection age to smaller value & try to use streaming API to verify if those settings actually work.